SU-SWS/sassdoc-theme-decanter
# Add sass

# READY FOR REVIEW

# Summary
This adds Sass and Grunt to the theme.
_See ticket:  https://github.com/SU-SWS/decanter/issues/558_

# Needed Review By (Date)
- Sprint 
- Soon

# Urgency
- Client is eagerly awaiting this.
- Housekeeping
- Fixes broken stuff
- Fixed on prod, need to put into code.
- New behat tests
- Not Urgent

# Steps to Test

- Checkout this branch
- Navigate to __Path__
- Verify you see __Text__

# Affected Projects or Products
- D8Core (https://github.com/SU-SWS/acsf-cardinalsites)

# Associated Issues and/or People
## Related JIRA ticket(s)

## Related PRs

## Accessibility checks performed

## More Information

## Folks to notify
`@sherakama` 
`@kerri-augenstein`

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
